  fun take_error(simple': NullablePointer[GSimpleAsyncResultT], error': NullablePointer[GErrorT]): None =>
    @g_simple_async_result_take_error(simple', error')
